mungegithub/run.sh

```diff
@@ -16,7 +16,7 @@
 
 while true; do
     /mungegithub \
-      --pr-mungers=blunderbuss,lgtm-after-commit,remove-needs-rebase \
```

Comment:
What's the correlation?  I'm missing somethign

On Mon, Aug 31, 2015 at 4:47 PM, Brendan Burns <notifications@github.com>
wrote:

> In mungegithub/run.sh
> <https://github.com/kubernetes/contrib/pull/43#discussion_r38373179>:
>
> > @@ -16,7 +16,7 @@
> >
> >  while true; do
> >      /mungegithub \
> > -      --pr-mungers=blunderbuss,lgtm-after-commit,remove-needs-rebase \
>
> yes, see: #45 <https://github.com/kubernetes/contrib/pull/45>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/contrib/pull/43/files#r38373179>.
>